catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/features/CatalogAppFeatures.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.content.monitor.features;
+
+import static org.ops4j.pax.exam.CoreOptions.maven;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.features;
+
+import java.util.Arrays;
+
+import org.ops4j.pax.exam.Option;
+
+public class CatalogAppFeatures {
+    public enum CatalogFeature {
```

Comment:
Same comment about a static array or resource bundle.

As it appears this model is going to be duplicated a lot here, a single resource bundle with several properties of comma-delimited strings would probably be the simplest solution.

If these are things that might be common across many component tests, then I would suggest pulling them out to a resource to be loaded by that abstract class I was mentioning might be worthwhile.

You could maintain a set of consts in the abstract class that the inheritors could reference; those consts would then be lookup keys into the resource bundle.